app/models/legal_aid_application.rb

```diff
@@ -61,4 +61,8 @@ def proceeding_type_codes_existence
   def create_app_ref
     self.application_ref = SecureRandom.uuid
   end
+
+  def set_consent_choice_date
```

Comment:
The attribute is `consent_choice_timestamp`, so the method should match - `set_consent_choice_timestamp`